pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/common/Felles.kt

```diff
@@ -80,4 +74,24 @@ object Felles {
                 English to antall.format() + " months"
             )
     }
+
+    data class DatoerOppramsningEllerListing(val foedselsdatoer: Expression<List<LocalDate>>): ParagraphPhrase<LangBokmalNynorskEnglish>(){
```

Comment:
DatoOppramsningEllerListe?

Reply:
Vanskelig å finne navn på dyret. Det er en frase som enten sier:
` bla og bla`

eller ved 3 eller flere
```
:
- bla
- bla
- bla
```

Reply:
Mente bare at "Datoer" og "Listing" var litt merkelig norsk i denne konteksten. Innså forøvrig nå at denne tingen egentlig kunne vært generell på `Expression<List<String>>`.

Reply:
Ikke helt fan av det. Da må vi passere med en string for ett tilfelle, og en liste av ting som blir oppramset om det er det andre tilfellet, og begge inneholder det samme. Da frister det mer å bare inline frasen. Kan vi løse dette om vi får problemet med at vi skal gjenbruke dette i andre tilfeller, og ikke nå?